src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,23 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+	
```

Comment:
Your comment says Contact is an abstract class. You have to use "abstract" keyword to declare a class as abstract.
This class looks more like a normal class. Yes, the purpose of inheritance can be still achieved with a normal class.